docusaurus/docs-internals/editor-controls/multivalue-controls/border.md

```diff
@@ -76,25 +91,44 @@ Returns an object with the following values:
 `width` - value which is applied for all sides of the border (changes when `"widthType"` is `"grouped"`); <br/>
 `widthType` - the current width type;
 
+Example of value:
+
+```js
+{
+  bottomWidth: 2,
+  hex: "#41a79c",
+  leftWidth: 2,
+  opacity: 1,
+  palette: "",
+  rigthWidth: 2,
+  style: "solid",
+  topWidth: 2,
+  width: 2,
+  widthType: "grouped"
```

Comment:
Check all keys in  `Border/converters.ts` on `fromElementModel` because as example instead of `opacity` should be
`colorOpacity`